akka-remote/src/main/resources/reference.conf

```diff
@@ -93,9 +93,9 @@ akka {
       canonical {
 
         # The default remote server port clients should connect to.
-        # Default is 20200, use 0 if you want a random available port
+        # Default is 25520, use 0 if you want a random available port
         # This port needs to be unique for each actor system on the same machine.
-        port = 20200
+        port = 25520
```

Comment:
Good to not keep it the default for the "old remoting" I guess. That's my only opinion. Unless we can make it 666 ofc. ;)